superset-frontend/cypress-base/cypress/integration/explore/control.test.ts

```diff
@@ -111,7 +111,7 @@ describe('VizType control', () => {
     // should load mathjs for line chart
     cy.get('script[src*="mathjs"]').should('have.length', 1);
     cy.get('script').then(nodes => {
-      expect(nodes.length).to.eq(numScripts);
+      expect(nodes.length).to.greaterThan(numScripts);
```

Comment:
I wasn't able to reproduce this locally (it was returning `nodes.length === 13`, `numScripts === 12`), but I noticed that this used to be `greaterThan` before this PR: #12218